iina/Base.lproj/Localizable.strings

```diff
@@ -260,6 +260,16 @@
 "osc_toolbar.music_mode" = "Enter music mode";
 "osc_toolbar.sub_track" = "Choose subtitle track";
 
+"mini_player.album_art" = "Toggle Album Art";
```

Comment:
I'd lean towards making this lowercase, unless you think there's a good reason why it should be capitalized?